targets/gnome

```diff
@@ -9,11 +9,32 @@ CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal evolution-data-server gnome-control-center \
-        gnome-screensaver gnome-session gnome-session-fallback \
-        gnome-shell gnome-themes-standard gvfs-backends nautilus \
-        unzip gtk2-engines-pixbuf
+check_distro() 
+{
+    ###list of distros that need gnome-session-fallback
+    set "squeeze" "wheezy" "kali" "trusty" "precise" 
+    name=$(lsb_release -cs)
+    #echo "$name"
+    for i do
+	if [ "$i" = "$name" ]; then
+	   echo "1" #The distro installed needs gnome-session-fallback
+	fi
+    done
+}
+fallback_need=$(check_distro)
 
```

Comment:
@divx118  thank you! Your solution is way more beautiful and the release function is definitely helpful and I was unawares of such utility. Thanks again, may I write a comment to credit you?